iot/manager/manager.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by the Apache 2.0
 // license that can be found in the LICENSE file.
 
-// Command manager lets you manage Cloud IoT Core devices and registries.
+// Package manager lets you manage Cloud IoT Core devices and registries.
```

Comment:
"Command" is OK. https://github.com/golang/go/wiki/CodeReviewComments#package-comments